src/lib/components/Societies/societies.json

```diff
@@ -0,0 +1,53 @@
+[
+    {
+      "continent": "🇪🇺 Europe",
+      "type": "contintent"
+    },
+    {
+      "name": "London Svelte Meetup",
+      "country": "🇬🇧 Great Britain",
+      "url": "https://www.meetup.com/svelte/"
+    },
+    {
+      "name": "Svelte Dublin",
+      "country": "🇮🇪 Ireland",
+      "url": "https://www.downtomeet.com/Svelte-Dublin"
+    },
+    {
+      "name": "Sveltejs Meetup Hamburg",
+      "country": "🇩🇪 Germany",
+      "url": "https://www.meetup.com/de-DE/Svelte-js-Meetup-Hamburg/",
+      "githuburl": "https://github.com/svelte-meetup-hamburg/meetups/issues"
+    },
+    {
+      "name": "Amsterdam SvelteJS",
+      "country": "🇳🇱 Netherlands",
+      "url": "https://www.meetup.com/Amsterdam-SvelteJS/"
+    },
+    {
+      "name": "Svelte Bergen",
+      "country": "🇳🇴 Norway",
+      "twitter": "@stephanevanraes",
+      "url": "https://www.downtomeet.com/Svelte-Bergen"
+    },
+    {
+      "name": "Svelte Society Stockholm",
+      "country": "🇸🇪 Sweden",
+      "twitter": "@kevmodrome",
+      "url": "https://www.downtomeet.com/Svelte-Society-Stockholm"
+    },
+    {
+      "continent": "South America",
+      "type": "contintent"
+    },
+    {
+      "name": "Svelte Brasil",
+      "country": "🇧🇷 Brazil",
+      "telegram": "https://t.me/sveltebrasil",
+      "twitter": "@sveltebrasil",
+      "url": "https://sveltebrasil.dev/",
+      "youtube": "https://www.youtube.com/channel/UCp8jamqJRGg86eMnewxjWng",
+      "githuburl": "https://github.com/svelte-brasil"
+    }
+  ]
```

Comment:
this line probably should not be indented to align with the opening bracket

Reply:
4577d44.
I think we should run prettier once over the whole project, I'll probably open up another PR for this when we got this finished (or you can do if you wan't).